README.md

```diff
@@ -46,29 +47,60 @@ use pest::Parser;
 struct IdentParser;
 
 fn main() {
-    let pairs = IdentParser::parse_str(Rule::ident, "abc123").unwrap_or_else(|e| panic!("{}", e));
+    let pairs = IdentParser::parse_str(Rule::ident_list, "abc123 def456").unwrap_or_else(|e| panic!("{}", e));
 
+    //Because ident_list is silent, the iterator will contain idents
```

Comment:
Papercut here, but I think a space after the `//` make the comment more elegant.